apps/src/Register.cpp

```diff
@@ -220,6 +220,7 @@ auto main(int argc, char* argv[]) -> int
         resample2->fixedImage = *results["fixedImage"];
         resample2->movingImage = moving->image;
         resample2->transform = compositeTfms->result;
+	resample2->forceAlpha = parsed.count("enable-alpha") > 0;	
```

Comment:
```suggestion
        resample2->forceAlpha = parsed.count("enable-alpha") > 0;	
```

Reply:
I think your editor is using tabs instead of spaces. This should be the correct indentation.